Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
         .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.29.0"),
         .package(url: "https://github.com/eneko/MarkdownGenerator.git", from: "0.4.0"),
         .package(url: "https://github.com/onevcat/Rainbow", from: "3.0.0"),
-        .package(url: "https://github.com/eneko/System.git", from: "1.0.0")
+        .package(url: "https://github.com/eneko/ProcessRunner.git", from: "1.1.0")
```

Comment:
Please update this dependency to prevent introducing future breaking changes:
```suggestion
        .package(url: "https://github.com/eneko/ProcessRunner.git", .upToNextMajor(from: "1.1.0"))
```